ddtrace/contrib/bottle/patch.py

```diff
@@ -1,12 +1,21 @@
 
 import os
 
+from ddtrace import config
+
 from .trace import TracePlugin
 
 import bottle
 
 import wrapt
 
+# Default settings
+config._add(
+    'bottle',
+    {'service_name': os.environ.get('DATADOG_SERVICE_NAME') or 'bottle'},
+)
```

Comment:
This shows how one could maintain compatibility with the envvar approach to setting service name for some services.